crates/matrix-sdk-base/src/rooms/normal.rs

```diff
@@ -174,6 +174,9 @@ pub struct Room {
     /// user has marked as seen so they can be ignored.
     pub seen_knock_request_ids_map:
         SharedObservable<Option<BTreeMap<OwnedEventId, OwnedUserId>>, AsyncLock>,
+
+    /// A sender that will notify receivers when room member updates happen.
+    pub room_member_updates_sender: broadcast::Sender<RoomMembersUpdateKind>,
```

Comment:
Since this is `pub`, it's part of the public API, and it must be tested that it works correctly when subscribing :smiling_imp: